app/assets/src/components/ui/icons/BacteriaCultureIcon.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const BacteriaCultureIcon = ({ className }) => {
```

Comment:
should we be naming these now `IconCultureBacteria` by the new naming schema?

Reply:
I would say `IconBacteriaCulture` would match the naming schema better. Specially since we already have `BacteriaIcon`. 
But now that you mentioned it, not sure if we should consider this an icon. I might need to review the whole placement of this component.
@happyimadesignr What would this component be in the design system?

Reply:
@tfrcarvalho @gregdingle We are calling these illustrations!  ImgBacteriaXlarge

The illustrations are on the bottom of this page in the design system with names: https://chanzuckerberg.invisionapp.com/share/UWVLU4OH37F#/401265432_1-7_Bases-_Icons_-_Illustrations